src/shared/seccomp-util.c

```diff
@@ -308,13 +318,27 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "lseek\0"
                 "pread64\0"
                 "preadv\0"
+                "preadv2\0"
                 "pwrite64\0"
                 "pwritev\0"
+                "pwritev2\0"
                 "read\0"
                 "readv\0"
                 "write\0"
                 "writev\0"
         },
+        [SYSCALL_FILTER_SET_CHOWN] = {
+                .name = "@chown",
+                .help = "Change ownership of files and directories",
+                .value =
+                "chown\0"
+                "chown32\0"
+                "fchown\0"
+                "fchown32\0"
+                "fchownat\0"
+                "lchown\0"
+                "lchown32\0"
+        },
```

Comment:
Did you let these inside `@privileged` set on purpose ? they should be removed from that set or at least have a way to import the `@chown` into `@privileged` and let the syscalls opencoded in one place. However, for backward compatibility, capabilities should cover this so removing them from `@privileged` should be ok.

P.S. will post patches to add logic to auto import a set into another one soon for the abstracted sandbox model.

Reply:
you are right, `@privileged` should just import this group. will fix.